itests/docker-compose.yml

```diff
@@ -10,16 +10,16 @@ services:
     networks:
       - net
```

Comment:
After you're done can you please run this file through a linter to fix all the different types of spacings